extensions/wikia/UserRenameTool/UserRenameToolProcess.class.php

```diff
@@ -844,11 +1062,20 @@ public function renameInTable( $dbw, $table, $uid, $oldusername, $newusername, $
 				$affectedRows = $dbw->affectedRows();
 				$this->addLog( "SQL: " . $dbw->lastQuery() );
 				$dbw->commit();
-				$this->addLog( "In {$dbName}.{$table}.{$extra['username_column']} {$affectedRows} row(s) was(were) updated." );
-				sleep( USERRENAME_LOOP_PAUSE );
+				$this->addLog(
+					"In {$dbName}.{$table}.{$extra['username_column']} {$affectedRows} row(s) was(were) updated."
+				);
+				sleep( 5 );
 			}
 		} catch ( Exception $e ) {
-			$this->addLog( "Exception in renameInTable(): " . $e->getMessage() . ' in ' . $e->getFile() . ' at line ' . $e->getLine() );
+			$this->addLog(
+				"Exception in renameInTable(): " .
+				$e->getMessage() .
+				' in ' .
+				$e->getFile() .
+				' at line ' .
+				$e->getLine()
+			);
```

Comment:
This might be more readable as a single message string via sprintf:
```php
$this->addLog( sprintf(
    "Exception in renameInTable(): %s in %s at line %s",
    $e->getMessage(),
    $e->getFile(),
    $e->getLine()
) )
```